canonical-web-and-design/charmhub.io
# 42 replace details data with context

## Done

- Replace details data with context

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to a charm (`/charm`) and to a bundle (`/charm?type=bundle`) details page and see it looks as before


## Issue / Card

Fixes #

## Screenshots

[if relevant, include a screenshot]
